### ISSUE TITLE
After Deleting all integrations, number of messages still shows up

### ISSUE THREAD

**Issue by snandakumar87** [syndesisio/syndesis#4013]
After Deleting all integrations, number of messages still shows up, technically these messages are not reachable, since they are tied to the integrations and integrations are no longer available

## Expected behavior
The messages related to the integrations deleted shouldnt show up on the summary page.

![image](https://user-images.githubusercontent.com/39506194/47867082-e3d61c80-ddd6-11e8-8b9a-533a30e4062a.png)

## Tasks involved / Steps to Reproduce

1. Create an integration, such that messages are populated
2. Delete the integration and navigate to the summary page
3. We can see that the number of integrations shows up as 0, but the messages still shows up as 55.
4. These messages arent reachable in any way after the integrations are deleted.

**Comment by heiko-braun** [comment]
@syndesisio/backend IIRC there is related report by @KurtStam already? What are you thoughts on this?

**Comment by KurtStam** [comment]
@heiko-braun The stats are coming from elastic. That data is not removed when an integration is deleted, the idea at the time was that if you re-import the same integration it keeps counting forward. This was as described by the User Story. As engineer I've always considered it weird, but we'd prob need to run it by @kcbabo / @gaughan before we can call it a bug.

**Comment by KurtStam** [comment]
sorry yes prometheus not elastic.

**Comment by chirino** [comment]
This is not a bug. This show what has happened in the system. Please note the integrations are never really deleted. They can be resurrected:)

**Comment by snandakumar87** [comment]
@chirino totally understand the idea behind doing it that way, but when looking at this for the first time, it shows a different understanding 🙂

**Comment by zregvart** [comment]
Then perhaps a small info like icon in the corner and a tooltip explaining this better would be good?

cc @syndesisio/uxd

**Comment by snandakumar87** [comment]
@zregvart +1 i think that will be a good visual cue too.

**Comment by gaughan** [comment]
This would be great, even a manual way to clear messages would be nice for demos etc

**Comment by dongniwang** [comment]
![screen shot 2019-02-05 at 4 32 37 pm](https://user-images.githubusercontent.com/24943812/52305667-afd53f00-2963-11e9-9eae-1aad1d4b9da1.png)

Would something like this be suffient? We have this pattern for Technology Preview. 

@zregvart @gaughan Please take a look and let me know. 

cc: @syndesisio/uxd

**Comment by heiko-braun** [comment]
@dongniwang I think this is appropriate.

**Comment by gaughan** [comment]
We need to be able to view the messages and clear them down.

**Comment by dongniwang** [comment]
@gaughan - What do you mean by viewing the message? I would assume that could be done someplace else other than in the pop-over. And we don't have an area in the app to manage all these messages. 

> We need to be able to view the messages and clear them down.

Maybe we can open a separate issue for this requirement? ^^

**Comment by gaughan** [comment]
So the current UI has the number of messages (55) If I have errors the next thing I want to do is see them..read them and maybe clear them. I can open a separate issue

**Comment by gaughan** [comment]
https://github.com/syndesisio/syndesis/issues/4461

**Comment by dongniwang** [comment]
Thanks @gaughan! 

Then who will be implementing this update? @heiko-braun 

What exactly does "Total Messages" mean? Also hoping to get @TovaCohen to help with the right wording for the pop-over.

**Comment by TovaCohen** [comment]
The doc  says:
Total number of messages that have been processed by integrations in this environment in the last 30 days. This includes messages that were processed by integrations that might no longer be running or that might have been deleted from this environment.

Is that too long for a popup? Here's a shorter version:
All messages processed in the last 30 days. Integrations that processed some of these messages might no longer be running or might have been deleted.

**Comment by dongniwang** [comment]
+1 on the shorter version. Thank you @TovaCohen!

**Comment by amysueg** [comment]
@gashcrumb Can you take a look at this to implement this message design for 7.3? And if so, open a separate implementation issue we can close this design issue?

FYI @mcoker
